lib/webenginesettings.cpp

```diff
@@ -129,6 +129,10 @@ void SailfishOS::WebEngineSettings::initialize()
     engineSettings->setPreference(QStringLiteral("browser.enable_automatic_image_resizing"),
                                   QVariant::fromValue<bool>(true));
 
+    // Enable user agent overrides
+    engineSettings->setPreference(QStringLiteral("general.useragent.updates.enabled"),
+                                  QVariant::fromValue<bool>(true));
```

Comment:
Should this be following `MGConfItem(QStringLiteral("/apps/sailfish-browser/settings/useragent_update_enabled"))`, or is that old behaviour now?

Reply:
I debugged that it's mandatory to have this set or the saved overrides are not read at all. Also this lib could be used by any app so using settings from sailfish-browser is a bit questionable.

Reply:
Yes, fair enough. It might be good to allow apps to set their own overrides, but that's something for the future.